Sources/SwifterSwift/UIKit/UIImageExtensions.swift

```diff
@@ -182,26 +182,40 @@ public extension UIImage {
         return newImage
     }
 
+    #if !os(watchOS)
     /// SwifterSwift: UIImage tinted with color
     ///
     /// - Parameters:
     ///   - color: color to tint image with.
     ///   - blendMode: how to blend the tint
     /// - Returns: UIImage tinted with given color.
     func tint(_ color: UIColor, blendMode: CGBlendMode) -> UIImage {
```

Comment:
Not related to the issue, but since we are here: Note that we work with an alpha 1.0 always, so would be a good improvement add it as parameter :)
So the function looks like `func tint(_ color: UIColor, blendMode: CGBlendMode, alpha: CGFloat = 1.0) -> UIImage`
It is important to let the alpha with default value 1.0 so we make it optional and don't break existing code ;)

Reply:
It makes sense to me, thanks for the suggestion ;)
I will update it.